### ISSUE TITLE
Schema validation errors generating Bicep Types for storageimportexport

### ISSUE THREAD

**Issue by anthony-c-martin** [Azure/azure-rest-api-specs#14504]
The following validations are flagging errors when running the latest autorest (`3.2.0`, core version `3.4.2`) for `specification/storageimportexport/resource-manager/readme.md`:

```
Executing: autorest --use=/home/runner/work/bicep-types-az/bicep-types-az/src/generator --azureresourceschema --output-folder=/tmp/_bcp_1621030621058/storageimportexport --multiapi /home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/readme.md
AutoRest code generation utility [cli version: 3.2.0; node: v14.16.1, max-memory: 2048 MB]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/home/runner/work/bicep-types-az/bicep-types-az/src/generator/node_modules/@autorest/core/dist' (3.4.2)
INFORMATION: > Loading local AutoRest extension '@autorest/bicep' (/home/runner/work/bicep-types-az/bicep-types-az/src/generator)
INFORMATION: > Loading AutoRest extension '@autorest/azureresourceschema' (~3.0.45->3.0.101)
Process() cancelled due to failure 
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/locations > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:50:10 ($.paths["/providers/Microsoft.ImportExport/locations"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/locations/{locationName} > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:92:10 ($.paths["/providers/Microsoft.ImportExport/locations/{locationName}"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:138:10 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:198:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:264:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > put > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:379:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].put.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > delete > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:447:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].delete.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > patch > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:317:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].patch.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}/listBitLockerKeys > post > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:496:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}/listBitLockerKeys"].post.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/operations > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:551:10 ($.paths["/providers/Microsoft.ImportExport/operations"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/locations > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:50:10 ($.paths["/providers/Microsoft.ImportExport/locations"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/locations/{locationName} > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:92:10 ($.paths["/providers/Microsoft.ImportExport/locations/{locationName}"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:138:10 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:198:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:264:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].get.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > put > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:379:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].put.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > delete > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:447:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].delete.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName} > patch > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:317:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}"].patch.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}/listBitLockerKeys > post > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:496:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ImportExport/jobs/{jobName}/listBitLockerKeys"].post.externalDocs.url)
ERROR: Schema violation: should match format "uri" (paths > /providers/Microsoft.ImportExport/operations > get > externalDocs > url)
  format: uri
    - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/storageimportexport/resource-manager/Microsoft.ImportExport/preview/2021-01-01/storageimportexport.json:551:10 ($.paths["/providers/Microsoft.ImportExport/operations"].get.externalDocs.url)
Exited with code 1
Failed to generate types for path 'storageimportexport'
```

**Comment by PhoenixHe-msft** [comment]
@anthony-c-martin Should be fixed by https://github.com/Azure/azure-rest-api-specs/pull/14278

**Comment by anthony-c-martin** [comment]
> @anthony-c-martin Should be fixed by #14278

Thanks @PhoenixHe-msft! Closing this issue as the PR has been merged.
